corehq/messaging/scheduling/views.py

```diff
@@ -1071,7 +1070,7 @@ def page_context(self):
                 "download_url": reverse("download_conditional_alert", args=(self.domain,)),
                 "adjective": _("SMS alert content"),
                 "plural_noun": _("SMS alert content"),
-                "help_link": "https://confluence.dimagi.com/display/commcarepublic/Bulk+download+and+upload+of+SMS+content+in+conditional+alerts", # noqa
+                "help_link": "https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143956488/Conditional+Alerts", # noqa
```

Comment:
This wasn't you, but ideally we should specify which checks flake8 is skipping:
```suggestion
                "help_link": "https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143956488/Conditional+Alerts",  # noqa: E501
```